resources/config-defaults.edn

```diff
@@ -338,13 +338,18 @@
  ;; nil = no limit
  :attachment-max-size 10000000
 
- ;; Should applications be deleted after a set period of time has passed?
- ;; Use map with key as required application state, and value as ISO-8601 duration:
- ;; {:application.state/draft "P90D" :application.state/closed "P7D"}
+ ;; Should applications be deleted after having been inactive for a set period of time?
+ ;; Use map with key as required application state, and value as a map:
+ ;;   {:application.state/draft {:delete-after "P90D" :reminder "P7D"}}
+ ;;   :delete-after - delete application after period of time since last activity
+ ;;   :reminder - (optional) send reminder about expiration to application members (as period of time before expiration)
```

Comment:
Isn't this called `:reminder-before` now?

Btw. this design limits us to only one reminder about expiration. But we can expand it later.